src/Common/CommonServices.ts

```diff
@@ -472,14 +481,66 @@ export interface IGlobalMessageBanner {
      * Links to supply to the format arguments in `messageFormat`
      */
     messageLinks?: IGlobalMessageLink[];
+
+    /**
+     * Where the banner should appear.
+     */
+    position?: string;
+}
+
+/**
+ * Toast Props with metadata to specify how it will be rendered by the service
+ */
+export interface IToast {
+    /**
+     * Optional text for the Call to Action
+     */
+    callToAction?: string;
+
+    /**
+     * Optional class name for the root toast element
+     */
+    className?: string;
```

Comment:
do we want to expose className? All they could do here is supply one of our first-party class names.